doc/config.toml

```diff
@@ -265,6 +262,10 @@ url = "http://www.apache.org/security/"
   version = "++version++ (Current)"
   url = "/docs/++version++/"
 
+[[params.versions]]
+version = "1.12.0"
+url = "https://avro.apache.org/docs/1.12.0/"
```

Comment:
Gives 404.
The HTMLs for the old (i.e. non-current) versions is hosted at SVN (I cannot find the location atm)

Reply:
maybe https://svn-master.apache.org/repos/asf/avro/site/publish/
found it at https://infra-reports.apache.org/#sitesource
but I don't remember now how it works ...

Reply:
there it is - https://svn-master.apache.org/repos/asf/avro/site/publish/docs/

Reply:
Yes, still have to get to that.

It looks like it is part of the release, but unsure how it works since we've changed build systems:

- https://avro.apache.org/docs/1.11.1/
- https://avro.apache.org/docs/1.10.0/

I'll dig into this and make sure to document it

Reply:
Ping me at Slack during EU working hours. I have some knowledge!

Reply:
I've uploaded 1.11.2 and 1.11.3 as well.